docs/reference/doofinder.cartitemstatsparams.title.md

```diff
@@ -4,10 +4,10 @@
 
 ## CartItemStatsParams.title property
 
-Title of the given item \*
+The title that describe the item. Usually, the title of item's page \*

```

Comment:
Since it's the third person, I suggest to change "describe" to "describes"

Reply:
done